frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetailView.unit.spec.tsx

```diff
@@ -273,7 +273,7 @@ function setup(
   );
 }
 
-describe("Object Detail", () => {
+describe("ObjectDetailView", () => {
```

Comment:
Modify the name so we don't get confused with the new test I created `frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.unit.spec.tsx`